static/app/views/replays/detail/console/index.tsx

```diff
@@ -73,6 +74,9 @@ function Console({breadcrumbs, startTimestampMs}: Props) {
         rowIndex={index}
       >
         <ConsoleLogRow
+          hasOccurred={
+            currentTime < relativeTimeInMs(item.timestamp || 0, startTimestampMs)
```

Comment:
tiny tiny nit, does `??` make more sense semantically?

Reply:
yeah, it's clearer that way. timestamp is required by replays, we need to know when on the timeline things happened... let me go fix that in the types that we use and this'll become moot.